packages/trader/src/Stores/Modules/Contract/Helpers/chart-marker-helpers.js

```diff
@@ -49,23 +49,10 @@ export const createMarkerStartTime = contract_info => {
 export const createMarkerSpotEntry = contract_info => {
     if (!contract_info.entry_tick_time) return false;
 
-    let marker_type = MARKER_TYPES_CONFIG.SPOT_ENTRY.type;
-    let component_props = {};
-
+    const marker_type = MARKER_TYPES_CONFIG.SPOT_ENTRY.type;
     const entry_tick = contract_info.entry_tick_display_value;
 
-    const spot_has_label = isDigitContract(contract_info.contract_type);
-    if (spot_has_label) {
-        marker_type = MARKER_TYPES_CONFIG.SPOT_MIDDLE.type;
```

Comment:
use this straightaway in the return instead of defining it 😄

Reply:
😄 👍🏼